Alignment/CommonAlignmentProducer/python/ALCARECOTkAlUpsilonMuMuPA_Output_cff.py

```diff
@@ -15,5 +19,25 @@
 )
 
 import copy
+# in Run3, SCAL digis replaced by onlineMetaDataDigis
+import copy
```

Comment:
same here

Reply:
I don't think it is really has any effect, but anyway I've removed the duplication in the last push.